journalwatch.py

```diff
@@ -364,15 +369,23 @@ def parse_config_files():
     return cfg, patterns
 
 
-def get_journal(since=None):
+def get_journal(since=None, priority=None):
     """Open the journal and get a journal reader.
 
     Args:
         since: A datetime object where to start reading.
                If None, the whole journal is read.
+        priority: A number between 1 and 7 for lowest priority
+               to consider.
+               If omited, defaults to "LOG_INFO", number 6.
     """
     j = journal.Reader()
-    j.log_level(journal.LOG_INFO)
+    if (int(priority) < 1):
+        j.log_level(1)
+    elif (int(priority) >= 1 and int(priority) <= 7):
+        j.log_level(int(priority))
+    else:
+        j.log_level(7)
```

Comment:
Instead of this if, you could simply do `priority = max(1, min(priority, 7))`. It might seem a bit of a hack, but it's a quite common thing.